project/frontend/src/components/ResultsMap.js

```diff
@@ -10,13 +10,19 @@ const mapStyle = {
 function MapContainer(props) {
   const [activeMarker, setActiveMarker] = useState({});
   const [showInfoWindow, setShowInfoWindow] = useState(false);
+  const [infoWindowNumber, setInfoWindowNumber] = useState('');
   // TODO: add a conditional to check if the list is empty
   const restaurant = props.restaurant;
   const matchCoords = restaurant[0].key.latLngCoords;
 
-  const onMarkerClick = (props, marker) => {
+  const onMouseOverMarker = (props, marker) => {
     setActiveMarker(marker);
     setShowInfoWindow(true);
+    setInfoWindowNumber(marker.id);
+  };
+
+  const onMouseOutMarker = (props) => {
```

Comment:
I don't think you need props as the param here if its not being used (not sure though and I don't think it's a problem to keep it anyway)